fiasko_bro/code_validator.py

```diff
@@ -126,14 +126,14 @@ def _run_warning_validators_until(self, failed_error_group_name, arguments):
             )
         return warnings
 
-    def run_validator_group(self, group, pre_validation=False, *args, **kwargs):
+    def run_validator_group(self, group, add_warnings=False, *args, **kwargs):
```

Comment:
Let's keep `add_warnings=False`, this will make the addition of the warnings more explicit.